src/erlcloud_ddb2.erl

```diff
@@ -3434,7 +3496,7 @@ tag_resource(ResourceArn, Tags) ->
                   -> tag_resource_return().
 tag_resource(ResourceArn, Tags, Config) ->
     erlcloud_ddb_impl:request(
-      Config,
+      Config, false,
```

Comment:
This should probably call the `/3` (`false` by default) version of the implementation, if you decide to go that route.